engine/src/main/scala/cromwell/webservice/routes/wes/RunListResponse.scala

```diff
@@ -1,19 +1,19 @@
 package cromwell.webservice.routes.wes
 
+import akka.http.scaladsl.model.StatusCodes
 import cromwell.services.metadata.MetadataService
 import cromwell.services.metadata.MetadataService.{WorkflowQueryFailure, WorkflowQuerySuccess}
 import cromwell.webservice.routes.wes.WesState.fromStatusString
 
-
 case class RunListResponse(runs: List[WesRunStatus], next_page_token: String)
 
 object RunListResponse {
-  def fromMetadataQueryResponse(response: MetadataService.MetadataQueryResponse): RunListResponse = {
+  def fromMetadataQueryResponse(response: MetadataService.MetadataQueryResponse): WesResponse = {
     response match {
       case w: WorkflowQuerySuccess =>
         val runs = w.response.results.toList.map(x => WesRunStatus(x.id, fromStatusString(x.status)))
-        RunListResponse(runs, "Not Yet Implemented")
-      case _: WorkflowQueryFailure => ??? // FIXME: What do we return when we fail to get a Workflow Query?
+        WesResponseRunList(runs)
+      case _: WorkflowQueryFailure => WesErrorResponse("Unable to fetch metadata query", StatusCodes.BadRequest.intValue)
```

Comment:
I assume this "query failure" object will contain a message or a reason? If so I think it's probably worth forwarding that to the API response